integration/spark2/src/main/scala/org/apache/spark/sql/CarbonDatasourceHadoopRelation.scala

```diff
@@ -195,8 +195,7 @@ case class CarbonDatasourceHadoopRelation(
   override def unhandledFilters(filters: Array[Filter]): Array[Filter] = new Array[Filter](0)
 
   override def toString: String = {
-    "CarbonDatasourceHadoopRelation [ " + "Database name :" + identifier.getDatabaseName +
-    ", " + "Table name :" + identifier.getTableName + ", Schema :" + tableSchema + " ]"
+    "carbondata"
```

Comment:
keep the name as "CarbonDatasourceHadoopRelation" instead of "carbondata". so that in plan we can know which class it is identified with.

Reply:
Isn't it a little odd if it shows "Scan CarbonDatasourceHadoopRelation DBNAME.TABLENAME" ?

Reply:
no we need to know what relation it is and also table name required.
@BJangir @ravipesala : please tell your opinion.

Reply:
Yes, table name and dbname are required otherwise it is difficult to know about which table relation it is in case of complex queries. So as @ajantha-bhat mentioned keep it as short like `Carbondata[dbname.tableName]`

Reply:
"Scan xxx DBNAME.TABLENAME" 

the upper text is generated by spark, not the changes I did. 
table name and dbname are already included.
Please review PR carefully. I also have snapshot for that in description.

Reply:
Also plan will be affected by this change, Plan also shows as just "carbondata" after this change.
@ravipesala : just "carbondata[dbname.tableName]" is enough or "CarbonDatasourceHadoopRelation[dbname.tableName]" need to keep ?

Reply:
@kevinjmh : check "explain extended" of the query, we don't see database name in plan now after this change I guess. please compare the plan before and after this change.

Reply:
have you test it? @ajantha-bhat 

my test show the plan shows correctly, not just "carbondata"

Reply:
@kevinjmh : ok, I checked. That line will not have db and table info, but other line will have. That is enough. 
now need to conclude whether to keep as "carbondata" or "CarbonDatasourceHadoopRelation"

Reply:
OK.  The reason I set it as "carbondata" because I test parquet table, it get the source shortName to print, "parquet". Actually `CarbonSource` also defines a shortName, but it used different logic. 

It's fine for me if you all have strong will to change the name to identify where this text from.

Reply:
Yes right.. I feel "CarbonDatasourceHadoopRelation" is better

Reply:
@ravipesala  what's your opinion?

Reply:
since no more vote, I am changing this to `CarbonDatasourceHadoopRelation`